coldfront/core/utils/management/commands/audit_data.py

```diff
@@ -0,0 +1,229 @@
+'''
+add a command audit_data to ensure that certain invariants hold. These include, but are not limited to:
+
+ Allocations have the expected start_date and end_date values.
+ Inactive Projects have "Expired" Allocations and zero SUs.
+ Each Project has at least one PI.
+ Users with a cluster UID should be associated with at least one Project (maybe).
+LRC only:
+ Each UserProfile that has access to Lawrencium should have a billing_activity.
+ Each Recharge Allocation that has access to Lawrencium should have a "Billing Activity"-typed AllocationAttribute.
```

Comment:
Can you limit line length to 80 characters? There's usually an option in your IDE to show you when you go over.